### PR TITLE
1525 accepted article peer reviewed

### LINKED ISSUE #1525: article.peer_review on acceptance
**Is your feature request related to a problem? Please describe.**
Editors sometimes forget to mark an article as reviewed and are surprised so see messages on article pages that inform readers that articles have not been reviewed.

**Describe the solution you'd like**
On acceptance:
- [x] Check if the article has had any completed reviews
- [x] Mark the article as reviewed
- [x] Leave the checkbox editable
- [x] On prepub, warn editors if it has not been marked as reviewed

### PR DESCRIPTION
**Pull request by ajrbyers**
- If an article has completed reviews we now mark it as reviewed automatically
- Add warnings to the prepub page including one for peer review

Closes #1525

### CODE REVIEW

**Comment by rhb123** [comment]
Thanks for this - I've just been assigned to this, is there anything further you need from me here?

**Comment by ajrbyers** [comment]
> Thanks for this - I've just been assigned to this, is there anything further you need from me here?

Nah I just assigned you so you can see its happening :P

File: src/templates/admin/journal/publish_article.html [hunk 1]
```diff
@@ -199,7 +211,7 @@ <h2>Confirm Article Set for Publication</h2>
         }
 
         data = {
-            'task_type': task_type,
+            'task_type': task_type,
```
Comment (mauromsl) [inline src/templates/admin/journal/publish_article.html:214]: There is a rogue whitespace here

Comment (ajrbyers) [reply src/templates/admin/journal/publish_article.html]: DELETED.

**Pull request merged**